source/cloud/windows.md

```diff
@@ -33,33 +38,68 @@ Click or select the `Overview` tab, you will see the Windows admin password
 
 ![image](img/windows_admin_pass.png)
 
-You should use this password to login the first time in your Windows machine.
-(You must reset the admin password to connect via Remote Desktop).
-
-## Reset Windows admin password
```

Comment:
maybe we should keep this section as default setup and tell the users to reset the admin password after the first boot as a good security advice.
The admin pass generated by tf is visible from the the metadata network as plain text, so visible to any user in the project and with access to the dashboard as well.